build/Build.Steps.Windows.cs

```diff
@@ -177,4 +179,31 @@ partial class Build
 
             AssemblyRedirectionSourceGenerator.Generate(netFxAssembliesFolder, generatedSourceFile);
         });
+
+    Target InstallNetFxAssembliesGAC => _ => _
+        .Unlisted()
+        .After(BuildTracer)
+        .OnlyWhenStatic(() => IsWin)
+        .Executes(() => RunNetFxGacOperation(install: true));
+
+    /// <remarks>
+    /// Warning: This target could cause potential harm to your system by removing a required library from GAC.
+    /// </remarks>
+    Target UninstallNetFxAssembliesGAC => _ => _
+        .Unlisted()
+        .After(BuildTracer)
+        .OnlyWhenStatic(() => IsWin)
+        .Executes(() => RunNetFxGacOperation(install: false));
+
+    private void RunNetFxGacOperation(bool install)
```

Comment:
nit: I would rather pass `string operation` that would be `"-i"` or `"-u"`. I do not like bool parameters😉 No need to address it.